honeybee_grasshopper_radiance/src/HB Annual Glare Metrics.py

```diff
@@ -0,0 +1,136 @@
+_
+- Glare Autonmy is a metric describing the percentage of occupied
+hours that each sensor is below the glare threshold.
+- Spatial Glare Autonomy is a metric describing the percentage of the sensor grid
+that is free glare according to the glare threshold and the target time. The sGA
+value is expressed as a percentage of area.
+
+-
+    Args:
+        _results: An list of annual Radiance result files from the "HB Imageless Annual
+            Glare" component (containing the .dgp files and the sun-up-hours.txt).
+            This can also be just the path to the folder containing these result
+            files.
+        _occ_sch_: An annual occupancy schedule as a Ladybug Data Collection or a HB-Energy
+            schedule object. This can also be the identifier of a schedule in
+            your HB-Energy schedule library. Any value in this schedule that is
+            0.1 or above will be considered occupied. If None, a schedule from
+            9AM to 5PM on weekdays will be used.
+        _threshold_: Threshhold for daylight autonomy (DA) in lux (default: 300).
+        _min_max_: A list for min, max illuminacne thresholds for useful daylight illuminance
+            in lux. (Default: (100, 3000)).
+        grid_filter_: The name of a grid or a pattern to filter the grids. For instance,
+            first_floor_* will simulate only the sensor grids that have an
+            identifier that starts with first_floor_. By default all the grids
+            will be processed.
+        mesh_: An optional list of Meshes that align with the meshes used in the imageless
+            annual glare recipe, which will be used to assign an area to each sensor.
+            If no mesh is connected here, it will be assumed that each sensor represents
+            an equal area to all of the others.
+        _target_time_: A minimum threshold of occupied time (eg. 95% of the time), above
+            which a given sensor passes and contributes to the spatial glare
+            autonomy. (Default: 95%).
+
+    Returns:
+        report: Reports, errors, warnings, etc.
+        GA: Glare autonomy results in percent. GA is the percentage of occupied hours
+            that each sensor is free of glare according to the DGP threshold.
+            Each value is for a different sensor of the grid. These can be plugged
+            into the "LB Spatial Heatmap" component along with meshes of the sensor
+            grids to visualize results.
+        sGA: Spatial glare autonomy as percentage of area for each analysis grid.
+        pass_fail: A data tree of zeros and ones, which indicate whether a given senor
+            passes the criteria for being free of glare (1) or fails the criteria (0). 
+            Each value is for a different sensor of the grid. These can be plugged
+            into the "LB Spatial Heatmap" component along with meshes of the
+            sensor grids to visualize results.
```

Comment:
I like this output but it would be helpful to clarify that being "being free of glare" does not mean there are no hours with DGP above the `_glare_thresh_`, right? It just means that the sGA has a percentage of glare-free hours less than the `_target_time_`, right?